### ISSUE TITLE
Provide a way to have a built-in default app.js with all the web-deps

### ISSUE THREAD

**Issue by ia3andy** [quarkiverse/quarkus-web-bundler#166]
This app would import all the web-dependencies so they are bundled.

I need to see how to deal with css/scss.

**Comment by ia3andy** [comment]
Here is the upstream issue: https://github.com/mvnpm/esbuild-java/issues/96

**Comment by ia3andy** [comment]
This is available in 1.4.0.CR1 using:
```properties
quarkus.web-bundler.dependencies.auto-import=all
```

or 
```properties
quarkus.web-bundler.dependencies.auto-import=styles
```
